rtengine/iimage.h

```diff
@@ -98,6 +98,10 @@ class ImageDatas : virtual public ImageDimensions
         return "unknown";
     }
 
+    virtual bool isBW () const
```

Comment:
Agreed. Though my workstation is a bit faster. ;)

Why isn't LTO enabled by default? I had positive experience with it at work, where we go for `-Os`.